sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample4_MultivariateDetect.cs

```diff
@@ -0,0 +1,204 @@
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Train error. {0}", e.Message));
+                throw new Exception(e.Message);
+            }
+        }
+        private async Task<DetectionResult> detectAsync(AnomalyDetectorClient client, string datasource, Guid model_id,DateTimeOffset start_time, DateTimeOffset end_time, int max_tryout = 500)
+        {
+            try
+            {
+                Console.WriteLine("Start detect...");
+                Response<Model> get_response = await client.GetMultivariateModelAsync(model_id).ConfigureAwait(false);
+
+                DetectionRequest detectionRequest = new DetectionRequest(datasource, start_time, end_time);
+                Response result_response = await client.DetectAnomalyAsync(model_id, detectionRequest).ConfigureAwait(false);
+                var ok = result_response.Headers.TryGetValue("Location", out string result_id_path);
+                Guid result_id = Guid.Parse(result_id_path.Split('/').LastOrDefault());
+                // get detection result
+                Response<DetectionResult> result = await client.GetDetectionResultAsync(result_id).ConfigureAwait(false);
+                int tryout_count = 0;
+                while (result.Value.Summary.Status != DetectionStatus.Ready & tryout_count < max_tryout)
+                {
+                    System.Threading.Thread.Sleep(2000);
+                    result = await client.GetDetectionResultAsync(result_id).ConfigureAwait(false);
+                    tryout_count += 1;
+                }
+
+                if (result.Value.Summary.Status != DetectionStatus.Ready)
+                {
+                    Console.WriteLine(String.Format("Request timeout after {0} tryouts", max_tryout));
+                    return null;
+                }
+
+                return result.Value;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Detection error. {0}", e.Message));
+                throw new Exception(e.Message);
+            }
+        }
+
+        private async Task exportAsync(AnomalyDetectorClient client, Guid model_id, string model_path = "model.zip")
+        {
+            try
+            {
+                Stream model = await client.ExportModelAsync(model_id).ConfigureAwait(false);
+                if (model != null)
+                {
+                    var fileStream = File.Create(model_path);
+                    model.Seek(0, SeekOrigin.Begin);
+                    model.CopyTo(fileStream);
+                    fileStream.Close();
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(String.Format("Export error. {0}", e.Message));
+                throw new Exception(e.Message);
+            }
+        }
+
+        private async Task deleteAsync(AnomalyDetectorClient client, Guid model_id)
```

Comment:
Add snippet region here. Check [this](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample1_DetectEntireSeriesAnomaly.cs#L22) for an example. You can also refer the docs [here](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets).